app/src/main/java/com/plaid/linksample/MainActivityJava.java

```diff
@@ -13,15 +13,15 @@
 import android.view.View;
```

Comment:
This is essentially the previous `MainActivityResultContractJava`, but with a the success/error case extracted into a separate function to keep things cleaner + with the options menu to switch to other examples.